services/venice-controller/src/test/java/com/linkedin/venice/controller/TestVeniceParentHelixAdmin.java

```diff
@@ -1465,6 +1454,26 @@ public void testStoreVersionCleanUpWithMoreVersions() {
     }
   }
 
+  private void mockControllerClients() {
+    Map<String, ControllerClient> controllerClientMap = new HashMap<>();
+
+    for (int i = 0; i < 3; i++) {
+      ControllerClient client = mock(ControllerClient.class);
+      StoreResponse storeResponse = new StoreResponse();
+      Store s = TestUtils.createTestStore("s" + i, "test", System.currentTimeMillis());
+      s.setCurrentVersion(i + 4); // child region current versions 4,5,6
```

Comment:
I am assuming you want the current version of `i` to be offset by the number of total stores (or total number of clients) in the test.
Would be nice to make that explicit to prevent potential overlap. e.g., changes the `for loop` condition to y but forgot to make changes to the ln:1464

e.g., instead of 3, can we extract it to some meaningful constant and set the current version in ln:1464 to 
`s.setCurrentVersion(i + <CONSTANT>)`.

Reply:
No, its the off by 1 mistake, its just setting the current version different in different region. I could make the regions number a constant